kernel/src/main/java/org/kframework/compile/CloseCells.java

```diff
@@ -10,281 +16,290 @@
 import org.kframework.kore.*;
 import org.kframework.utils.errorsystem.KEMException;
 
-import java.util.ArrayList;
-import java.util.HashSet;
-import java.util.List;
-import java.util.Set;
-
-import static org.kframework.kore.KORE.*;
-
 /**
  * Remove any use of dots in cells, by replacing them with variables and appropriate connectives.
  * This expects parent cells to have been added by earlier passes, it will only add variables
  *
- * The input to this pass is represents cells as described by {@link IncompleteCellUtils}.
- * In the output cells no longer have dots. Leaf cells have a single argument which is
- * the body, and parent cells are applied directly to the child cells and variables
- * as arguments of the KApply (though not necessarily in the right order) as
- * expected by {@link SortCells}.
+ * <p>The input to this pass is represents cells as described by {@link IncompleteCellUtils}. In the
```

Comment:
What is the purpose of `<p>` here?
Was it added automatically? Or you added this manually?

Reply:
Added automatically; the JavaDoc coding standards say that to separate paragraphs in the _output_ from the tool, you should add a `<p>` tag like this. It's basically a question of the difference between what we see here in the input (comment) and the output (JavaDoc HTML).